third_party/python/Python/pylifecycle.c

```diff
@@ -462,8 +477,10 @@ _Py_InitializeEx_Private(int install_sigs, int install_importlib)
     if (install_sigs)
         initsigs(); /* Signal handling stuff, including initintr() */
 
+#ifdef USE_TRACEMALLOC
     if (_PyTraceMalloc_Init() < 0)
```

Comment:
I would recommend writing these statements as:

```c
if (IsModeDbg() && _PyTraceMalloc_Init() < 0)
```

See https://github.com/jart/cosmopolitan/blob/master/libc/dce.h

Reply:
I've used `#ifdef MODE_DBG` for this and `_PyTraceMalloc_Fini`, which isn't in a conditional.